stor/tests/test_integration_s3.py

```diff
@@ -20,24 +20,29 @@ class S3IntegrationTest(BaseIntegrationTest.BaseTestCases):
     bucket on S3.
 
     In order to run the tests, you must have valid AWS S3 credentials set in the
-    following environment variables: AWS_DEFAULT_REGION, AWS_ACCESS_KEY_ID,
-    AWS_SECRET_ACCESS_KEY. In order to have the tests actually run, you also need
-    to additionally set the AWS_TEST_ACCESS_KEY_ID environment variable. For now,
-    it can simply be set to any value.
+    following environment variables: AWS_TEST_ACCESS_KEY_ID,
+    AWS_TEST_SECRET_ACCESS_KEY (and optionally AWS_DEFUALT_REGION).
```

Comment:
typo.  Normally I'd let it slide but it's an env variable name so I must be a pedant.